sdk/src/builder.rs

```diff
@@ -432,6 +447,29 @@ impl Builder {
                 //println!("adding resource {}", id);
                 builder.resources.add(id, data)?;
             }
+
+            // Load the c2pa_manifests.
+            // We add the manifest data to any ingredient that has a matching active_manfiest label.
+            if file.name().starts_with("manifests/") && file.name() != "manifests/" {
+                let mut data = Vec::new();
+                file.read_to_end(&mut data)?;
+                let manifest_label = file
+                    .name()
+                    .split('/')
+                    .nth(1)
+                    .ok_or(Error::BadParam("Invalid manifest path".to_string()))?;
```

Comment:
It's worth checking out `ZipFile::encode_name()` to get back a `Path`.

Reply:
Thanks, I'll take a look at that.

Reply:
do you mean enclosed_name()? The main purpose of that is to prevent attacks when unzipping files into folders, and we aren't doing that here. I think it is actually simpler to process this as a string since we need to convert the underscores back into colons anyway.